Sources/FoundationExtensions/Dictionary+Extensions.swift

```diff
@@ -72,7 +83,7 @@ extension Dictionary {
     ///   - rhs: Another dictionary to merge.
     /// - Returns: An dictionary with keys and values from both.
```

Comment:
I know it's not from this PR, but just saw this:

```suggestion
    /// - Returns: A dictionary with keys and values from both.
```

Reply:
Oops thanks.